composer.json

```diff
@@ -34,7 +34,11 @@
         "phpunit/phpunit": "^9.5",
         "roave/infection-static-analysis-plugin": "^1.16",
         "spatie/phpunit-watcher": "^1.23",
-        "vimeo/psalm": "^4.18"
+        "symfony/console": "^6.0",
+        "symfony/process": "^6.0",
+        "vimeo/psalm": "^4.18",
+        "vishnubob/wait-for-it": "dev-master",
```

Comment:
Did it run without errors? I looked at the Dockerfile and did not see any differences that would allow using this script and not falling on Alpine

Reply:
It was just a draft, there is no need for this package at all.